item-seller/readme.md

```diff
@@ -37,16 +47,40 @@ WORLD_ADDRESS=
 
 #ERC20 TOKEN ADDRESS COPIED FROM ITEM SELLER DEPLOYMENT
 ERC20_TOKEN_ADDRESS=
+
+#DONT NEED TO CHANGE IF YOUR RUNNING LOCALLY
+SSU_ID=
 ```
 
 You can adjust the remaining values in the .env file as needed, though they are optional.
 
 
+<details markdown="block">
+<summary>Changing optional environment values</summary>
+
+#### Setting item, price and payment address
+You can set the item you want to sell, the address that recieves payments and the price in Wei. 10^18 wei is equal to one Ether.
```

Comment:
can we give some examples here 
say : 1 lens is 5 token then the value is : 5 * 10^18 
10 lens is 5 tokens then the value is : 5 * 10 ^ 17

Reply:
Have done.